scripts/requirements-gateway.txt

```diff
@@ -17,6 +17,7 @@ google-cloud-storage
 cachetools
 paramiko
 rich
+typer
```

Comment:
Where does the gateway require typer?

Reply:
For setting the usage [here](https://github.com/skyplane-project/skyplane/blob/f5cafda0024f1169494e8229d20c97b01430a42e/skyplane/api/usage/client.py#L164-L193)

Reply:
Since this is only used in the client, use dependency injection like this: https://github.com/skyplane-project/skyplane/blob/35cb10e9b96a81b9918b6eaf29867cabb9bdbbba/skyplane/api/impl/tracker.py#L63-L64

We can remove this dependency from the Docker image then